recipe/conda_build_config.yaml

```diff
@@ -0,0 +1,10 @@
+# sycl runtime components depends on 2.28 glibc.
+c_stdlib_sycl_version:  # [linux]
+  - 2.28                # [linux]
+c_stdlib_version:       # [linux]
+  - 2.17                # [linux]
```

Comment:
In particular, I don't understand why the sycl components would affect MKL? And if it's just for building, then it should be enough to bump the image version, but leave `c_stdlib_version` untouched.